src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2463,42 +2463,18 @@ def Var(self, num):
 
     @property
     def math(self):
-        """APDL math interface
+        """Interface to launch PyAnsys Math from PyMAPDL."""
+        import ansys.math.core.math as pymath
 
-        Returns
-        -------
-        :class:`MapdlMath <ansys.mapdl.core.math.MapdlMath>`
-
-        Examples
```

Comment:
I think the deleted docstring still add value. Although we are using a different package, I should still get some information when I do `help(mapdl.math)`.

Further more, to avoid instantiate an `AnsMath` class every time I call this, you could just:
```py
        if self._math is None:
            import ansys.math.core.math as pymath
            self._math = pymath.AnsMath(self)
        return self._math
```
You need to create `self._math` in `__init__` method. We do that for other classes (like `Krylov`).
I understand this is just replicating how it was done, but I think it is better to instantiate this class only once. What do you think?

Reply:
Agreed. Unless we really need to start froma  clean sheet every time we perform a mathematical operation (which I hope not), let's reuse the same class instance.